packages/react-router/src/route.ts

```diff
@@ -1056,11 +1056,7 @@ export class Route<
     this._to = fullPath as TrimPathRight<TFullPath>
   }
 
-  addChildren<
```

Comment:
I need to undo this change and possible add a new internal method to add children only for file based